capa/engine.py

```diff
@@ -200,6 +221,16 @@ def evaluate(self, ctx):
         raise ValueError("cannot evaluate a subscope directly!")
 
 
+def get_statement_description(children):
```

Comment:
this is probably the most appropriate place to document how we're extracting descriptions from statement nodes. recommend that we explain our lessons learned so far.